README.md

```diff
@@ -36,6 +36,45 @@ Install package via PyPI
 pip3 install dxfeed
 ``` 
 
+## Installation from sources
+
+Reminder: initialize and pull git submodule after cloning the repo:
+```bash
+git submodule init
+git submodule update
+``` 
+
+To install dxfeed from source you need Poetry, Cython and taskipy in addition to the normal dependencies above.
+Poetry provides a custom installer. This is the recommended way of installing poetry according to
+[documentation](https://python-poetry.org/docs/)
+
+For osx / linux / bashonwindows:
+
+```bash
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+```
+
+Cython and taskipy can be installed from pypi:
+
+```python
+pip3 install cython taskipy
+``` 
+
+In the project root directory (same one where you found this file after
+cloning the git repo), execute:
+
+```bash
+task build
+pip install dist/dxfeed-x.x.x.tar.gz
+```
+
+or for installing in [development mode](https://pip.pypa.io/en/latest/reference/pip_install.html#editable-installs):
+
+```bash
+task build_inplace
+pip install -e .
+```
```

Comment:
It's worth to mention `poetry run task clear` somewhere. 
Execution of `build` after some changes and `build_inplace` could confuse some users. 😕

Reply:
Task clear is not a necessary option. If any pyx files are changed build_inplace will recompile them. Clear and build are used for dev purposes. I do not think these commands should be mentioned on the front page